R/VaribleChartsIndviduals.R

```diff
@@ -69,19 +66,12 @@ VaribleChartsIndviduals <- function(jaspResults, dataset, options){
     jaspGraphs::geom_rangeframe() +
     jaspGraphs::themeJaspRaw()
 
-  ppPlot$plotObject <-  p
-  return(ppPlot)
-}
-.MRchart <- function(dataset, options, variable){
-  title <- gettextf("Variable: %s", variable )
-  ppPlot <- createJaspPlot(width = 700, height = 350, title = title)
-  ppPlot$dependOn(optionContainsValue = list(variables = variable))
-
+  #Moving rage chart
   #data
-  data <- data.frame(process = dataset[[.v(variable)]])
-  xmr.raw.r <- matrix(cbind(data$process[1:length(data$process)-1], data$process[2:length(data$process)]), ncol=2)
+  data2 <- data.frame(process = dataset[[.v(variable)]])
+  xmr.raw.r <- matrix(cbind(data2$process[1:length(data2$process)-1], data2$process[2:length(data2$process)]), ncol=2)
```

Comment:
if you try this you get:
```r
x <- 1:5
1:length(x) - 1
# 0 1 2 3 4
```
are you sure you want `1:length(data2$process)-1` and not `1:(length(data2$process)-1)`?